www/src/Pages/AddonsConfigPage.js

```diff
@@ -1271,6 +1322,163 @@ export default function AddonsConfigPage() {
+									error={errors.tiltLeftAnalogLeftPin}
+									isInvalid={errors.tiltLeftAnalogLeftPin}
+									onChange={handleChange}
+									min={-1}
+									max={29}
+								/>
+								<FormControl type="number"
+									label="Tilt Left Analog Right Pin"
+									name="tiltLeftAnalogRightPin"
+									className="form-select-sm"
+									groupClassName="col-sm-3 mb-3"
+									value={values.tiltLeftAnalogRightPin || -1}
+									error={errors.tiltLeftAnalogRightPin}
+									isInvalid={errors.tiltLeftAnalogRightPin}
+									onChange={handleChange}
+									min={-1}
+									max={29}
+								/>
+							</Row>
+
+							<Row class="mb-3">
+								<FormControl type="number"
+									label="Tilt Right Analog Up Pin "
+									name="tiltRightAnalogUpPin"
+									className="form-select-sm"
+									groupClassName="col-sm-3 mb-3"
+									value={values.tiltRightAnalogUpPin || -1}
+									error={errors.tiltRightAnalogUpPin}
+									isInvalid={errors.tiltRightAnalogUpPin}
+									onChange={handleChange}
+									min={-1}
+									max={29}
+								/>
+								<FormControl type="number"
+									label="Tilt Right Analog Down Pin"
+									name="tiltRightAnalogDownPin"
+									className="form-select-sm"
+									groupClassName="col-sm-3 mb-3"
+									value={values.tiltRightAnalogDownPin || -1}
+									error={errors.tiltRightAnalogDownPin}
+									isInvalid={errors.tiltRightAnalogDownPin}
+									onChange={handleChange}
+									min={-1}
+									max={29}
+								/>
+								<FormControl type="number"
+									label="Tilt Right Analog Left Pin"
+									name="tiltRightAnalogLeftPin"
+									className="form-select-sm"
+									groupClassName="col-sm-3 mb-3"
+									value={values.tiltRightAnalogLeftPin || -1}
+									error={errors.tiltRightAnalogLeftPin}
+									isInvalid={errors.tiltRightAnalogLeftPin}
+									onChange={handleChange}
+									min={-1}
+									max={29}
+								/>
+								<FormControl type="number"
+									label="Tilt Right Analog Right Pin"
+									name="tiltRightAnalogRightPin"
+									className="form-select-sm"
+									groupClassName="col-sm-3 mb-3"
+									value={values.tiltRightAnalogRightPin || -1}
+									error={errors.tiltRightAnalogRightPin}
+									isInvalid={errors.tiltRightAnalogRightPin}
+									onChange={handleChange}
+									min={-1}
+									max={29}
+								/>
+							</Row>
+							<Row class="mb-3">
+								<FormSelect
+									label="Tilt SOCD Mode"
+									name="tiltSOCDMode"
+									className="form-select-sm"
+									groupClassName="col-sm-3 mb-3"
+									value={values.tiltSOCDMode}
+									error={errors.tiltSOCDMode}
+									isInvalid={errors.tiltSOCDMode}
+									onChange={handleChange}
+								>
+									{TILT_SOCD_MODES.map((o, i) => <option key={`button-tiltSOCDMode-option-${i}`} value={o.value}>{o.label}</option>)}
+								</FormSelect>
+							</Row>
+						</div>
+						<FormCheck
+							label="Enabled"
```

Comment:
`{t('Common:switch-enabled')}` here

Reply:
Thank you for your advice. I have successfully resolved the merge conflicts and updated the code to include i18n support. All text strings have now been localized.

Reply:
Thank you for your advice. I have successfully resolved the merge conflicts and updated the code to include i18n support. All text strings have now been localized.